crates/sui-core/src/consensus_adapter.rs

```diff
@@ -471,6 +466,46 @@ impl ConsensusAdapter {
         )
     }
 
+    // According to the throughput profile we want to either allow some transaction duplication or not)
+    // When throughput profile is Low and the validator is in position = 1, then it will submit to consensus with much lower latency.
+    // When throughput profile is High then we go back to default operation and no-one co-submits.
+    fn override_by_throughput_profiler(&self, position: usize, latency: Duration) -> Duration {
+        const LOW_THROUGHPUT_LATENCY_MS: u64 = 1250;
```

Comment:
In the prod setup this is currently set to 1.5s - setting this to a bit lower to allow more duplication.